src/mon/MonmapMonitor.cc

```diff
@@ -761,6 +763,8 @@ bool MonmapMonitor::prepare_command(MonOpRequestRef op)
     pending_map.remove(name);
     pending_map.disallowed_leaders.erase(name);
     pending_map.last_changed = ceph_clock_now();
+    dout(20) << "removing mon." << name << " at " << addrs
+       << ", there will be " << pending_map.size() << " monitors" << dendl;
```

Comment:
If you want this output in the log, you should dump a copy from the stringstream that renders it in the next line instead of re-generating it. :) (ie, ```dout(20) << ss.str() << dendl;``` I think should do it.)